File: Coding Challenges/Daily Challenge/2026/January/20260111-Par-For-the-Hole.js
function golfScore(par, strokes) {
  switch(strokes) {
    case 1: return "Hole in one!";
    case par - 2: return "Eagle";
    case par - 1: return "Birdie";
    case par: return "Par";
    case par + 1: return "Bogey";
    case par + 2: return "Double bogey";
  }
  
  return strokes;
}

File: Coding Challenges/Daily Challenge/2026/January/20260111-Par-For-the-Hole.py
def golf_score(par, strokes):
    if strokes == 1:
        return "Hole in one!"
    elif strokes == par - 2:
        return "Eagle"
    elif strokes == par - 1:
        return "Birdie"
    elif strokes == par:
        return "Par"
    elif strokes == par + 1:
        return "Bogey"
    elif strokes == par + 2:
        return "Double bogey"

    return strokes